pulser/devices/_device_datacls.py

```diff
@@ -47,31 +47,31 @@ class Device:
     _channels: Tuple[Tuple[str, Channel], ...]
     interaction_coeff: float = 5008713.
 
-    def __post_init__(self):
+    def __post_init__(self) -> None:
         # Hack to override the docstring of an instance
         self.__dict__["__doc__"] = self._specs(for_docs=True)
 
     @property
-    def channels(self):
+    def channels(self) -> Dict:
```

Comment:
You can be a bit more specific here:
```suggestion
    def channels(self) -> Dict[str, Channel]:
```